dotcom-rendering/src/components/StickyLiveblogAsk.importable.tsx

```diff
@@ -0,0 +1,309 @@
+	flex-direction: column;
+	align-items: flex-start;
+	padding: ${space[1]}px;
+	gap: ${space[1]}px;
+`;
+const bodySection = css`
+	color: ${palette.neutral[0]};
+	${textSans14}
+`;
+const ctaSection = css`
+	color: ${palette.neutral[0]};
+	margin-top: ${space[3]}px;
+	margin-bottom: ${space[4]}px;
+`;
+const cta = css`
+	left: ${space[2]}px;
+	min-width: 100%;
+	width: 100%;
+	min-height: 30px;
+	height: 30px;
+`;
+const buttonStyles = {
+	textPrimary: palette.neutral[7],
+	backgroundPrimary: palette.brandAlt[400],
+	backgroundPrimaryHover: palette.brandAlt[300],
+};
+
+const contributionsTheme = {
+	button: buttonStyles,
+	link: buttonStyles,
+};
+
+interface StickyLiveblogAskProps {
+	referrerUrl: string;
+	shouldHideReaderRevenueOnArticle: boolean;
+}
+
+const whatAmI = 'sticky-liveblog-ask';
+
+export const StickyLiveblogAsk: ReactComponent<StickyLiveblogAskProps> = ({
+	referrerUrl,
+	shouldHideReaderRevenueOnArticle,
+}) => {
+	const { renderingTarget } = useConfig();
+	const countryCode = useCountryCode(whatAmI);
+	const pageViewId = usePageViewId(renderingTarget);
+
+	const ABTestAPI = useAB()?.api;
+
+	// We can check if a user is in a variant, returns a boolean
+	// ABTestTest being an ab test that was passed in via the ab test array
+	const userInVariant = ABTestAPI?.isUserInVariant(
+		'StickyLiveBlogAskTest',
+		'variant',
+	);
+
+	// should we show ourselves?
+	const [showSupportMessagingForUser, setShowSupportMessaging] =
+		useState<boolean>(false);
+	const authStatus = useAuthStatus();
+
+	useEffect(() => {
+		const isSignedIn =
+			authStatus.kind === 'SignedInWithOkta' ||
+			authStatus.kind === 'SignedInWithCookies';
+		setShowSupportMessaging(!shouldHideSupportMessaging(isSignedIn));
+	}, [authStatus]);
+
+	// tracking
+	const tracking: Tracking = useMemo(() => {
+		return {
+			ophanPageId: pageViewId ?? '',
+			platformId: 'GUARDIAN_WEB',
+			clientName: 'dcr',
+			referrerUrl,
+			// message tests
+			abTestName: whatAmI,
+			abTestVariant: 'control',
+			campaignCode: whatAmI,
+			componentType: 'ACQUISITIONS_OTHER',
+		};
+	}, [pageViewId, referrerUrl]);
+
+	const urlWithRegionAndTracking = addRegionIdAndTrackingParamsToSupportUrl(
+		baseUrl,
+		tracking,
+		undefined,
+		countryCode,
+	);
+
+	// ophan tracking
+	const [hasBeenSeen, setNode] = useIsInView({
+		debounce: true,
+	});
+
+	useEffect(() => {
+		if (hasBeenSeen) {
+			// For ophan
+			void submitComponentEvent(
+				createViewEventFromTracking(tracking, tracking.campaignCode),
+				renderingTarget,
+			);
+		}
+	}, [hasBeenSeen, tracking, renderingTarget]);
+
+	const onButtonCtaClick = () => {
+		void submitComponentEvent(
+			createClickEventFromTracking(tracking, tracking.campaignCode),
+			renderingTarget,
+		);
+	};
+	const canShow =
+		userInVariant &&
+		showSupportMessagingForUser &&
+		!shouldHideReaderRevenueOnArticle;
+
+	return (
+		<>
+			{canShow && (
+				<div css={stickyLeft} ref={setNode}>
+					<div css={imageHeader}>
+						<svg
```

Comment:
as well as any optimisation of the svg itself I'd say this should be moved to its own component because it's pretty hard to scroll past!

Reply:
Moved the image into a separate const (and optimised thanks to Mateusz).